pysat_requirements.txt

```diff
@@ -0,0 +1,8 @@
+dask
+netCDF4
```

Comment:
```suggestion
netCDF4<1.5.7
```

Reply:
since this is working currently, I wonder if the ICON/GOLD files do not have the compression issue? Or is it only for writing files?